java/ql/src/Security/CWE/CWE-094/ArbitraryApkInstallation.qhelp

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE qhelp PUBLIC
+ "-//Semmle//qhelp//EN"
+ "qhelp.dtd">
+<qhelp>
+  <overview>
+    <p>
+      Android allows an application to install an APK (Android package kit)
+      using an <code>Intent</code> with
+      the <code>"application/vnd.android.package-archive"</code> MIME type. If
+      the file used in the <code>Intent</code> is from a location that is not
+      controlled by the application (for example, the SD card which is
+      universally writable), this can result in the installation of an
+      application which was not intended.
```

Comment:
```suggestion
      controlled by the application (for example, an SD card that is
      universally writable), this can result in the unintended installation of untrusted applications.  
```

Minor suggestions